ftw/simplelayout/browser/client/web/js/app/simplelayout/Block.js

```diff
@@ -49,6 +49,13 @@ define([
       return this;
     };
 
+    this.markLinks = function() {
+      var url = window.location.protocol + "//" + window.location.host;
+      // all http links (without the link-plain class), not within this site
+      $("a[href^=\"http\"]:not(.link-plain):not([href^=\"" + url + "\"])", this.element).attr("target", "_blank");
```

Comment:
Fair enough, didn't see that.

Did you think about just calling Plone's function? What was your conclusion?
Something like:
```javascript
if (typeof "scanforlinks" === "function") {
    scanforlinks();
}
```